src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelEvents.cs

```diff
@@ -114,6 +115,22 @@ public virtual void OnRequestShowHideGallery(bool showGallery)
             }
         }
 
+        public event RequestOpenDocumentationLinkHandler RequestOpenDocumentationLink;
+        public virtual void OnRequestOpenDocumentationLink(OpenDocumentationLinkEventArgs e)
+        {
+            // let any registered documentation link handlers process the request
+            if (RequestOpenDocumentationLink != null) RequestOpenDocumentationLink(e);
+            // if no handlers are registered, we fall back to handling the link with the default operating system handler
+            else Process.Start(new ProcessStartInfo(e.Link.AbsoluteUri));
```

Comment:
hmm, I'm not sure about this seems easy to abuse.

Reply:
removed the fallback, now silently ignored if no handler is registered. also left code comment around this decision for future
90ca3425ce8e6b6e67134246ca58327900367dc3